language/move-prover/doc/user/spec-lang.md

```diff
@@ -35,77 +41,80 @@ framework documentation][FRAMEWORK] which has specifications embedded.
         - [Pragma Inheritance](#pragma-inheritance)
         - [General Pragmas and Properties](#general-pragmas-and-properties)
     - [Pre and Post State](#pre-and-post-state)
     - [Helper Functions](#helper-functions)
         - [Uninterpreted Functions](#uninterpreted-functions)
+        - [Axioms](#axioms)
     - [Let Bindings](#let-bindings)
-    - [Aborts-If Condition](#aborts-if-condition)
-        - [Aborts-If Condition With Code](#aborts-if-condition-with-code)
-    - [Aborts-With Condition](#aborts-with-condition)
-    - [Requires Condition](#requires-condition)
-    - [Ensures Condition](#ensures-condition)
-    - [Modifies Condition](#modifies-condition)
-    - [Invariant Condition](#invariant-condition)
-        - [Struct Invariants](#struct-invariants)
-        - [Global Invariants](#global-invariants)
-            - [Isolated Global Invariants](#isolated-global-invariants)
-            - [Modular Verification and Global Invariants](#modular-verification-and-global-invariants)
-    - [Assume and Assert Conditions in Code](#assume-and-assert-conditions-in-code)
-    - [Specification Variables](#specification-variables)
-    - [Schemas](#schemas)
-        - [Basic Schema Usage](#basic-schema-usage)
-        - [Schema Expressions](#schema-expressions)
-        - [Schema Apply Operation](#schema-apply-operation)
-    - [Opaque Specifications](#opaque-specifications)
-    - [Abstract Specifications](#abstract-specifications)
-    - [Documentation Generation](#documentation-generation)
+        - [Aborts-If Condition](#aborts-if-condition)
+            - [Aborts-If Condition With Code](#aborts-if-condition-with-code)
+        - [Aborts-With Condition](#aborts-with-condition)
+        - [Requires Condition](#requires-condition)
+        - [Ensures Condition](#ensures-condition)
+        - [Modifies Condition](#modifies-condition)
+        - [Invariant Condition](#invariant-condition)
+            - [Function Invariants](#function-invariants)
+            - [Struct Invariants](#struct-invariants)
+            - [Global Invariants](#global-invariants)
+                - [Isolated Global Invariants](#isolated-global-invariants)
+                - [Modular Verification and Global Invariants](#modular-verification-and-global-invariants)
+        - [Assume and Assert Conditions in Code](#assume-and-assert-conditions-in-code)
+        - [Specification Variables](#specification-variables)
+        - [Schemas](#schemas)
+            - [Basic Schema Usage](#basic-schema-usage)
+            - [Schema Expressions](#schema-expressions)
+            - [Schema Apply Operation](#schema-apply-operation)
+        - [Opaque Specifications](#opaque-specifications)
+        - [Abstract Specifications](#abstract-specifications)
+        - [Documentation Generation](#documentation-generation)
 - [Expressiveness](#expressiveness)
 - [Appendix: Experimental/Deprecated Features](#appendix-experimentaldeprecated-features)
```

Comment:
I noticed that this section does not exist now, so maybe we could remove it in the table of content
```suggestion
```

Reply:
Good catch.